app/main/views/users.py

```diff
@@ -76,15 +82,18 @@ def list_users():
 
         user_query = user_query.filter(User.supplier_id == supplier_id)
 
-    users = user_query.paginate(
+    paginated_users = user_query.paginate(
         page=page,
         per_page=current_app.config['DM_API_SERVICES_PAGE_SIZE'],
     )
 
     return jsonify(
-        users=[user.serialize() for user in users.items],
+        users=[user.serialize() for user in paginated_users.items],
+        meta={
+            "total_pages": paginated_users.pages
```

Comment:
@risicle added it [here](https://github.com/alphagov/digitalmarketplace-api/pull/424/files).  After doing some when we were indexing services as part of making G8 live, I think it would be good to have `total`s in all of our `list_*` routes.